scripts/bin/auto-install

```diff
@@ -1,70 +1,54 @@
 #!/bin/sh
 
-# log to SNAP_COMMON and TTY
-log() {
-    printf '%s %s\n' \
-        "$(date -Iseconds)" "$1" | tee -a "${SNAP_COMMON}/auto-install.log"
-}
-
-# Check if drive is mounted
-check_drive() {
-
-    while :; do
-        _watch_file="$SNAP_COMMON/watched"
-
-        log "Signal file is ${_watch_file}."
+mklog() {
+    # FIXME: if mklog() is called within a subshell,
+    #        then LOG_DIR isn't getting globally updated
+    _log_dir="${1:-"${LOG_DIR:="$(mktemp -d)"}"}"
```

Comment:
We never call `mklog` with any args; is this a forward-looking thing and if so, what is the intention?